packages/components/src/disabled/index.js

```diff
@@ -52,30 +53,50 @@ class Disabled extends Component {
 			attributes: true,
 			subtree: true,
 		} );
+
+		this.node.addEventListener( 'click', this.catchEvents );
```

Comment:
What if the preview is displayed inside a Button. will this prevent the "click" from happening?

Reply:
Good catch! I've removed `stopPropagation`.